spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -168,7 +168,7 @@ public ManagedChannel stackdriverSenderChannel() {
 
   @Bean(REPORTER_BEAN_NAME)
   @ConditionalOnMissingBean(name = REPORTER_BEAN_NAME)
-  public Reporter<Span> stackdriverReporter(
```

Comment:
To be certain that this will work, you should also add `ZipkinAutoConfiguration` to the existing `@AutoConfigureBefore(BraveAutoConfiguration.class)`. This will ensure that your auto-configuration defines its reporter before `ZipkinAutoConfiguration` attempts to do so.

Reply:
done.